### DISCUSSION TITLE
Ability to modify colorbar/legend name?

### DISCUSSION THREAD

**Discussion by rcarson3** [Alpine-DAV/ascent#1420] [Q&A]
Hi,

I was wondering if there was a way to modify the colorbar / legend names for a given field? I currently have a number of history variables for different materials that I'd like to plot and the names look something like: `MaterialName_Region#_HistoryVariableName` which isn't the most pleasant... Ideally, I'd like to do something similar to what I might do in Visit or paraview and be able to modify the annotation name associated with the colorbar/legend. I couldn't find any documentation related to this so I'm guessing this might not be currently possible?

Thanks!

**Comment by cyrush** [comment]
We don't support this yet, turning this into an issue ticket so we can explore this.
